spec/models/ems_cluster_spec.rb

```diff
@@ -213,4 +213,30 @@
       expect(result).to be_falsey
     end
   end
+
+  context "#tenant_identity" do
+    # admin user is needed to process Events
```

Comment:
Correct, but it is not being used "to process events".  I do not think the comment line is needed anymore as the use of `admin` in the tests below is clear.